install-gui.sh

```diff
@@ -67,17 +67,17 @@ do_install_npm_locally(){
     export N_PREFIX=${SCRIPT_DIR}/.n
     PATH="${N_PREFIX}/bin:$(npm bin):${PATH}"
```

Comment:
Is `npm bin` still valid in node18?

Reply:
@paninaro I verified `npm bin` was removed in v9.0.0. It is suggested to use `npm exec foo` or `npx --yes foo`.

Reply:
@ChiaMineJP Can you take a look at the issue @wallentx mentioned? Feel free to make whatever changes you feel are necessary.

Reply:
I've replaced `$(npm bin)` with `$(npm prefix)/node_modules/.bin`.
Reference:
https://docs.npmjs.com/cli/v8/commands/npm-prefix
https://docs.npmjs.com/cli/v9/configuring-npm/folders?v=true

Reply:
This actually may be proper, since the only way to reach this part of the code is if you have an `npm` version less than `v9.x.x`